base_layer/core/src/base_node/states/block_sync.rs

```diff
@@ -124,6 +163,9 @@ impl BlockSyncStrategy {
         sync_peers: &mut Vec<NodeId>,
     ) -> StateEvent
     {
+        shared.info = StatusInfo::BlockSync(BlockSyncInfo::new(None, None, None));
+
+        shared.status_event_publisher.send(shared.info.clone()).await;
```

Comment:
⚠️ 
It will only error if there are no subscribers, so ok to ignore.
```suggestion
        let _ = shared.status_event_publisher.send(shared.info.clone()).await;
```